packages/connect-core/package.json

```diff
@@ -7,15 +7,23 @@
   "scripts": {
     "build": "yarn compile",
     "dev": "yarn build --watch",
+    "test": "is-ci \"test:coverage\" \"test:watch\"",
```

Comment:
I decide to keep the same scripts on every project and in the global scope. Let me know what you think.